test/draw/gpu/BlitTest.ooc

```diff
@@ -0,0 +1,81 @@
+/* This file is part of magic-sdk, an sdk for the open source programming language magic.
+ *
+ * Copyright (C) 2016 magic-lang
+ *
+ * This software may be modified and distributed under the terms
+ * of the MIT license.  See the LICENSE file for details.
+ */
+
+use base
+use geometry
+use draw-gpu
+use draw
+use opengl
+use unit
+
+BlitTest: class extends Fixture {
+	sourceImage := RasterMonochrome fromAscii(
+		"< *X>
+		<XXXXXX  >
+		<X*  *X  >
+		<X*  *X  >
+		<X* XXX  >
+		<X**XXX  >
+		<XXXXXX  >"
```

Comment:
Both input image and output image has to be aligned for the GPU rendering to work. There might be an incorrect stride assumption in the GPU image constructors.